.github/workflows/release.yml

```diff
@@ -97,6 +97,11 @@ jobs:
         node-version-file: ".nvmrc"
         registry-url: 'https://registry.npmjs.org'
 
+    - name: Update NPM
+      run: |
+        npm install -g npm@latest
```

Comment:
it may cause npm inconsistent with other jobs, could we upgrade other npm as well?